repos/system_upgrade/el7toel8/actors/setuptargetrepos/actor.py

```diff
@@ -44,6 +44,10 @@ def process(self):
                 if repo_map.from_id in enabled_repos:
                     rhel_repos.append(RHELTargetRepository(repoid=repo_map.to_id))
 
+        for task in self.consume(RepositoriesSetupTasks):
+            for repo in task.to_enable:
+                rhel_repos.append(RHELTargetRepository(repoid=repo))
```

Comment:
repository in PES data is not the same format as repoid from rhsm, so unless this is taken care of later and I missed it you'll need to translate these repositories to repoids

Reply:
Implemented a very basic mapping from PES repository to RHSM repoid. As the comment in code explains, this should be considered temporary since it is not complete and needs to be updated manually in code. By now, we only have the same entries as RepositoriesMapping actor. Adding new ones can leak (possible) sensitive data. Is it possible that PES provides RHSM repoid directly?

Reply:
I want to discuss whether or not to provide it in PES at cabal